eZ/Bundle/EzPublishIOBundle/EzPublishIOBundle.php

```diff
@@ -28,6 +28,9 @@ public function build(ContainerBuilder $container)
                 $extension->getBinarydataHandlerFactories()
             )
         );
+        $container->addCompilerPass(
```

Comment:
Nitpick: no need to wrap this one. The one above was wrapped because it had long arguments.